lib/modules/dosomething/dosomething_settings/dosomething_settings.strongarm.inc

```diff
@@ -52,14 +52,14 @@ function dosomething_settings_strongarm() {
       'checkbox' => 1,
       'report-only' => 1,
```

Comment:
I'd like to give the report-only mode another one shot and see how it changes compared to https://github.com/DoSomething/phoenix/issues/5783#issuecomment-169110035.
Just in case we need to allow one of whitelist domains in another one rule or we simply forgot of something important.